code/src/index.css

```diff
@@ -5,9 +5,188 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+/******** SEARCH SECTION ********/
+
+.search-wrapper {
+  margin: auto;
+  padding-top: 40px;
+  display: flex;
+  justify-content: center;
+  border-bottom: 2px solid white;
```

Comment:
One suggestion would be to move this styling to the .search-field instead to make it look more aligned :)